Sources/Commands/Utilities/SymbolGraphExtract.swift

```diff
@@ -71,11 +72,14 @@ public struct SymbolGraphExtract {
         if includeSPISymbols {
             commandLine += ["-include-spi-symbols"]
         }
-        if emitExtensionBlockSymbols {
-            commandLine += ["-emit-extension-block-symbols"]
+        
+        let extensionBlockSymbolsFlag = emitExtensionBlockSymbols ? "-emit-extension-block-symbols" : "-omit-extension-block-symbols"
+        if DriverSupport.checkSupportedFrontendFlags(flags: [extensionBlockSymbolsFlag.trimmingCharacters(in: ["-"])], fileSystem: fileSystem) {
+            commandLine += [extensionBlockSymbolsFlag]
         } else {
-            commandLine += ["-omit-extension-block-symbols"]
+            print("warning: dropped \(extensionBlockSymbolsFlag) flag because it is not supported by this compiler version")
```

Comment:
We should probably pass down `swiftTool.observabilityScope` and use that here instead of `print`.

Reply:
Done in cca49d50e9df8f92c2f81f972a1cb604d59dc31f.